### ISSUE TITLE
What is the best practice for serving fonts/images via sails lift?

### ISSUE THREAD

**Issue by davidjconnolly** [balderdashy/sails#3101]
Hey guys, awesome work!  I'm really excited about the structure sails.js has and continues to bring to the node community :)

I'm having a really hard time figuring out how to serve up static assets in Sails 0.11.  I've got my assets/js & style folders concatenating and minifying to production.js/css respectively which is great.  However, how do I serve up say a Fonts and Images folder and add that to the assets pipeline?  Is there a best practice for this?

**Comment by mattlo** [comment]
To my knowledge, if you run `grunt`, it'll keep propagating your `/assets` to `/.tmp` directory (which is the *real* public directory for static files)

**Comment by sailsbot** [comment]
Thanks for posting, @davidjconnolly.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
